stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Events.kt

```diff
@@ -0,0 +1,90 @@
+package io.getstream.chat.docs.kotlin
+
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.channel.ChannelClient
+import io.getstream.chat.android.client.channel.subscribeFor
+import io.getstream.chat.android.client.events.ChatEvent
+import io.getstream.chat.android.client.events.ConnectedEvent
+import io.getstream.chat.android.client.events.ConnectingEvent
+import io.getstream.chat.android.client.events.DisconnectedEvent
+import io.getstream.chat.android.client.events.NewMessageEvent
+import io.getstream.chat.android.client.events.NotificationAddedToChannelEvent
+import io.getstream.chat.android.client.events.UserPresenceChangedEvent
+import io.getstream.chat.android.client.subscribeFor
+import io.getstream.chat.android.client.subscribeForSingle
+import io.getstream.chat.android.client.utils.observable.Disposable
+
+class Events(val client: ChatClient, val channelController: ChannelClient) {
+
+    fun listenSpecificChannelEvents() {
+        // Subscribe for new message events
+        val disposable: Disposable = channelController
+            .subscribeFor<NewMessageEvent> { newMessageEvent ->
+                // to get the message
+                val message = newMessageEvent.message
```

Comment:
I think it would be useful if the code here was exactly the same as what's in the docs, copy-pasted. These lines differ a bit from the current version of the docs, I'd update one of these to make them match.

This applies to a lot of these snippets, not just this specific one.

Reply:
Need some adjustment, can not exactly the same as what's in the docs cause some snippet code in the web is not updated/working like [this one ](https://getstream.io/chat/docs/event_typing/?language=kotlin) `channel.keystroke()`, I think the right code is `channel.keystroke().enqueue()`.

And @JcMinarro also suggests adding an example code that can be done inside a callback, instead of just an empty callback like in the web now.

Reply:
Yup, that's a good example of what we should update on the web. I'm mostly asking _when_ we'll update those. Perhaps it would be easier to do it when we add it to the repo, instead of all at once later.